### ISSUE TITLE
ScirptHookV Core Error While Executing RealVR.asi

### ISSUE THREAD

**Issue by qisuqi** [LukeRoss00/gta5-real-mod#335]
Hellooo, this is my first time using a mod so please excuse potential stupid questions. 

So I have checked and the ScriptHookV.dll is the most updated version (1.0.2215.0) but the game cannot load and gives me the error in the title.  

I have followed all the steps in the Quick Step, unrared the files to where GTA.exe is is and along with the three scripts in ScriptHookV_1.0.2215.0\bin. I have also added in TrainerV.asi and trainerv from the most updated Simple Trainer. 

So I'm really stuck now of what do to do, pretty please could someone help? 

p.s. my GTA is from the Epic store so not sure if that has something to do with it...

**Comment by spaztazticii** [comment]
i am sadly experiencing this same issue, i have played Boneworks quite smoothly with Virtual Desktop on Oculus Quest 2, and im not quite sure what part i've done wrong, from a fresh install, i've followed the guidelines. any input would be appreciated.

**Comment by LukeRoss00** [comment]
@qisuqi Double check that you are using Release 7 of the mod. Also, the updated `ScriptHookV.dll` must be extracted to the same folder as `GTA5.exe`, not to a `ScriptHookV_1.0.2215.0\bin` subfolder where the game would not find it.

@spaztazticii Virtual Desktop is not compatible with my mod, you need to use a USB cable.

**Comment by qisuqi** [comment]
I did all that, so all the files from both the mod and ScirtHookV were extracted where GTA.exe is, it still didn't work. 

I ended up unintall the game and re-did everything and now everything works! I think my problem the first time was due to RealConfig.bat file couldn't locate the setting.xml (even though I did that step manually and thought it worked). 

Anyway, thank you for the mod, it looks sick!
